fastlane/spec/actions_specs/set_github_release_spec.rb

```diff
@@ -4,7 +4,7 @@
       it "sends and receives the right content from GitHub" do
         stub_request(:post, "https://api.github.com/repos/czechboy0/czechboy0.github.io/releases").
           with(body: "{\"tag_name\":\"tag33\",\"name\":\"Awesome Release\",\"body\":\"Bunch of new things :+1:\",\"draft\":false,\"prerelease\":false,\"target_commitish\":\"test\"}",
-          headers: { 'Authorization' => 'Basic MTIzNDVhYmNkZQ==', 'Host' => 'api.github.com:443', 'User-Agent' => 'fastlane-set_github_release' }).
+          headers: { 'Authorization' => 'Basic MTIzNDVhYmNkZQ==', 'Host' => 'api.github.com:443', 'User-Agent' => 'fastlane-github_api' }).
```

Comment:
Are there any problems that User agents are modified?
(I think there are no problems.)

Or you can add `user_agent` option to `github_api` action to set it for just in case.